.github/workflows/registry.yml

```diff
@@ -92,14 +84,24 @@ jobs:
         run: |
           rsync -r $LEGACY_HELM_CHART $HELM_CHARTS_DIR
           rsync -r $HELM_CHART/* $LEGACY_HELM_CHART --exclude=Chart.yaml --exclude=crds
+      - uses: oprypin/find-latest-tag@v1
+        with:
+          repository: oam-dev/kubevela
+          releases-only: true
+        id: latest_tag
       - name: Tag helm chart image
         run:  |
-          version=${{ steps.get_version.outputs.VERSION }}
-          sed -i "s/latest/$version/g" $HELM_CHART/values.yaml
-          sed -i "s/latest/$version/g" $LEGACY_HELM_CHART/values.yaml
-          number=${version#"v"}
-          sed -i "s/0.1.0/$number/g" $HELM_CHART/Chart.yaml
-          sed -i "s/0.1.0/$number/g" $LEGACY_HELM_CHART/Chart.yaml
+          latest_tag=${{ steps.latest_tag.outputs.tag }}
+          tag=${GITHUB_REF#refs/tags/}
+          chart_version=${tag#"v"}
+          if [[ ${GITHUB_REF} == "refs/heads/master" ]]; then
+            tag=latest
+            chart_version=${latest_tag#"v"}-latest
```

Comment:
great job!  Let's do it better:
1. the latest version, let's add one for the minor version. for example, the current version is `0.3.2` then, we should  use `0.4.0-latest-master`
2. make sure the user can only install it by using the flag `--devel`
3. align the installation doc

Reply:
Done.